packages/sync-actions/src/utils/combine-validity-actions.js

```diff
@@ -0,0 +1,19 @@
+export default function combineValidityActions(actions = []) {
+  const [setValidFromAction, setValidUntilAction] = actions.filter(
```

Comment:
I could refactor this a bit

```
const combinedSetValidDatesAction = [
  'setValidFrom',
  'setValidTo'
]

const isOneOfCombined = actionName => combinedSetValidDatesAction.includes(actionName)
```

Then you will notice that this whole function could be a bit simpler